modules/ossm-updating-istio-control-plane-with-revisionbased-istiorevisiontag.adoc

```diff
@@ -0,0 +1,153 @@
+default-v1-24-3   Local   True    Healthy   True     v1.24.3   10m
+default-v1-24-4   Local   True    Healthy   True     v1.24.4   66s
+----
+
+.. Confirm that `IstioRevisionTag` resource is ready by running the following command:
++
+[source,terminal]
+----
+$ oc get istiorevisiontag
+----
++
+.Example Output
++
+[source,yaml]
+----
+NAME      STATUS    IN USE   REVISION          AGE
+default   Healthy   True     default-v1-24-4   10m44s
+----
+
+. Confirm that there are two control plane pods ready for each revision by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -n istio-system
+----
+
+. Confirm that the proxy sidecar version is the same by running the following command:
++
+[source,terminal]
+----
+$ istioctl proxy-status
+----
++
+.Example Output
++
+[source,yaml]
+----
+NAME                                                    CLUSTER        CDS                LDS                EDS                RDS                ECDS        ISTIOD                      VERSION
+details-v1-7d775cb4f6-5t9zm.bookinfo                    Kubernetes     SYNCED (2m25s)     SYNCED (2m25s)     SYNCED (2m17s)     SYNCED (2m25s)     IGNORED     istiod-default-v1-24-3-c98fd9675-r7bfw     1.24.3
+productpage-v1-7c4b6b857-mxrw6.bookinfo                 Kubernetes     SYNCED (2m35s)     SYNCED (2m35s)     SYNCED (2m17s)     SYNCED (2m35s)     IGNORED     istiod-default-v1-24-3-c98fd9675-r7bfw     1.24.3
+ratings-v1-5b896f8544-r552l.bookinfo                    Kubernetes     SYNCED (2m21s)     SYNCED (2m21s)     SYNCED (2m17s)     SYNCED (2m21s)     IGNORED     istiod-default-v1-24-3-c98fd9675-r7bfw     1.24.3
+reviews-v1-746f96c9d4-9pw8k.bookinfo                    Kubernetes     SYNCED (2m17s)     SYNCED (2m17s)     SYNCED (2m17s)     SYNCED (2m17s)     IGNORED     istiod-default-v1-24-3-c98fd9675-r7bfw     1.24.3
+reviews-v2-97bdf5876-4mzx5.bookinfo                     Kubernetes     SYNCED (2m35s)     SYNCED (2m35s)     SYNCED (2m17s)     SYNCED (2m35s)     IGNORED     istiod-default-v1-24-3-c98fd9675-r7bfw     1.24.3
+reviews-v3-77d9db6844-djgjk.bookinfo                    Kubernetes     SYNCED (2m19s)     SYNCED (2m19s)     SYNCED (2m17s)     SYNCED (2m19s)     IGNORED     istiod-default-v1-24-3-c98fd9675-r7bfw     1.24.3
+----
++
+The `VERSION` column should match the old control plane version.
+
+. Restart the application workloads so that the new version of the sidecar gets injected by running the following command:
++
+[source,terminal]
+----
+$ oc rollout restart deployment -n bookinfo
+----
+
+.Verification
+
+. Verify that the new version of the sidecar is running by entering the following command:
++
+[source,terminal]
+----
+$ istioctl proxy-status
+----
++
+The `VERSION` column should match the new control plane version.
+
+. Verify the deletion of the old control plane, `{istio}`, and `IstioRevision` resources.
+
+.. Verify the deletion of the old control plane by running the following command:
```

Comment:
"deletion of" sounds a bit odd to me. Maybe consider wording these like "Verify that the old control plane has been deleted by..."